xml/art_sle_ha_install_quick.xml

```diff
@@ -715,15 +715,20 @@ softdog                16384  1</screen>
     situation. If you have not set up your fencing mechanism correctly, the cluster
     will not work  properly.</para>
    <para>Before using the cluster in a production environment, test it thoroughly
-    according to your use cases.
+    according to your use cases or with the help of the &cmd.test.script;
+    script.
     <!-- toms 2016-08-01: For SP3, refer to the "Testing Cluster Guide" or
      to Lars Pinnes iptable rules
     -->
    </para>
-    <remark>toms 2016-07-27: Fate#321073
+   <sect2 xml:id="sec-ha-inst-quick-test-resource-failover">
+    <title>Testing Resource Failover</title>
+    <para>
+     As a quick test, the following procedure checks on resource failovers:
```

Comment:
The procedure is not run automatically, but requires human interaction, therefore better phrase in active voice:
' To check resource failover, execute the following steps:' (or similar)

Reply:
Ok, done.